README.md

```diff
@@ -47,4 +47,4 @@ Implement the ability to edit a todo title on double click:
 
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_todo-app-with-api/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https:/IrynaMariiko00.github.io/react_todo-app-with-api/) and add it to the PR description.
```

Comment:
The demo link in the PR description is wrong now, add link from this file 
<img width="1286" alt="image" src="https://github.com/user-attachments/assets/c63fa33f-3232-444c-b3b1-b4c6d49dfdbe" />